docs/B-Computations/a-simple-calculation.md

```diff
@@ -154,24 +162,26 @@ To store the area in memory involves 4 program instructions:
 The multiplication operation takes the form:
 
 ```c
- operand * operand
+operand * operand
 ```
 
-operand is a placeholder for the variable or constant being multiplied.  **`*`** denotes the 'multiply by' operation.  The value of this expression is equal to the result of the multiplication. 
+_**operand**_ is a placeholder for the variable or constant being multiplied.  `*` denotes the 'multiply by' operation.  The value of this expression is equal to the result of the multiplication. 
 
 ### Assignment
 
 The assignment operation stores the value of an expression in the memory location of the destination variable.  Assignment takes the form:
 
 ```c
- destination = expression
+destination = expression
 ```
 
-_**destination**_ is a placeholder for the destination variable.  _**expression**_ refers to the value to be assigned to the destination variable.  = denotes the 'is assigned from' operation.  We call **`=`** the assignment operator. 
+_**destination**_ is a placeholder for the destination variable.  _**expression**_ refers to the value to be assigned to the destination variable.  = denotes the 'is assigned from' operation.  We call `=` the assignment operator. 
```

Comment:
`=` should be surrounded by backticks. 

![image](https://user-images.githubusercontent.com/86167097/141406896-79a16549-4fd6-4eb5-a10c-b3cdfb4cf13f.png)